lib/web_ui/lib/src/engine/skwasm/skwasm_impl/paragraph.dart

```diff
@@ -102,8 +103,30 @@ class SkwasmParagraph implements ui.Paragraph {
   bool get didExceedMaxLines => paragraphGetDidExceedMaxLines(handle);
 
   @override
-  void layout(ui.ParagraphConstraints constraints) =>
+  void layout(ui.ParagraphConstraints constraints) {
     paragraphLayout(handle, constraints.width);
+    if (!_hasCheckedForMissingCodePoints) {
```

Comment:
I don't believe it can. The reason is that you can't actually change the list of font families on the text styles or paragraph style once it is a paragraph, that's specified at the `ParagraphBuilder` stage. Once you've already created the paragraph, the only way to add more fallback fonts to it is by actually rebuilding the paragraph with new text styles.